### DISCUSSION TITLE
添加按键监听，声道分离。直接取代carplay

### DISCUSSION THREAD

**Discussion by gotoded** [eiyooooo/Easycontrol_For_Car#49] [Ideas]
一，易控能够监听汽车按键，自定义按键功能。
二，分离开导航，媒体，电话声音通道。
大胆设想如果是一个安卓盒子，例如：各种电视TV盒子。其实就是目前市面上的转换盒子）。
那么闲置的盒子，手机。刷个通电自启，直接秒杀那些垃圾车载盒子。

**Comment by eiyooooo** [comment]
每台车机能获得的权限都不一样，汽车按键也没有一个统一的定义，很难做按键监听的，也可以试着用tasker配合易控的广播控制来实现
